src/main/java/com/dougnoel/sentinel/filemanagers/FileManager.java

```diff
@@ -98,4 +104,60 @@ public static String getClassPath(String className) {
 			return null;
 		}
 	}
+	
+	/**
+	 * Saves an image of type File to the given sub-directory of the configured, or default, image path.
+	 * 
+	 * @param subDirectory the sub-directory within the configured or default image path.
+	 * If the sub-directory is null, the root image directory will be used.
+	 * @param imageFileName the file name of the image to save.
+	 * @param imageFile the File of the image to save.
+	 */
+	public static void saveImage(String subDirectory, String imageFileName, File imageFile) throws IOException {
+		if(subDirectory == null) 
+			subDirectory = "";
+		else
+			subDirectory += "/";
+		
+		File destinationFile = new File(Configuration.imageDirectory() + "/" + subDirectory + imageFileName);
+    	FileUtils.copyFile(imageFile, destinationFile);
+	}
+	
+	/**
+	 * Saves an image of type BufferedImage to the given sub-directory of the configured, or default, image path.
+	 * 
+	 * @param subDirectory the sub-directory within the configured or default image path.
+	 * If the subDirectory is null, the root image directory will be used.
```

Comment:
This line should go in the description above.

Reply:
I rewrote the javadoc comments to put the null optional information higher up in a few places in FileManager